src/mediaplayer.cs

```diff
@@ -20,7 +20,7 @@
 using UIKit;
 #endif
 using System;
-#if WATCH
+#if WATCH // TODO Will be removed after CoreMedia and AVFoundation are updated
 using AVMediaSelectionGroup = Foundation.NSObject;
 using AVMediaSelectionOption = Foundation.NSObject;
 #endif
```

Comment:
^ that creates unusable API and it's too easy to forget
Better not update `watchOS-MediaPlayer.todo` so it stays visible (and under our radar).

IOW just do the iOS parts until other parts are ready

Reply:
Okay I will do that!